usage/rest-server/src/main/java/brooklyn/rest/security/PasswordHasher.java

```diff
@@ -24,8 +24,9 @@
 
 public class PasswordHasher {
     public static String sha256(String salt, String password) {
-        byte[] salted = Charsets.UTF_8.encode((salt == null ? "" : salt) + password).array();
-        HashCode hash = Hashing.sha256().hashBytes(salted);
+        if (salt == null) salt = "";
+        byte[] bytes = (salt + password).getBytes(Charsets.UTF_8);
```

Comment:
Throwing would seem appropriate, or defaulting to the empty string, as is done for the salt. Perverse usage, but should at least be consistent.